examples/data-objects/prosemirror/src/fluidBridge.ts

```diff
@@ -350,21 +350,21 @@ export class ProseMirrorTransactionBuilder {
     }
 
     private getOpenStart(node: IProseMirrorNode[]): number {
-        if (!node || node.length === 0) {
+        if (node === undefined || node.length === 0) {
             return 0;
         }
 
         const start = node[0];
-        return !start._open || !start.content ? 0 : 1 + this.getOpenStart(start.content);
+        return !(start._open ?? false) || !start.content ? 0 : 1 + this.getOpenStart(start.content);
```

Comment:
This looks wrong -- should it maybe be something like this?  Similar question about risk of `null` though.

```suggestion
        return (start._open === undefined || start.content === undefined) ? 0 : 1 + this.getOpenStart(start.content);
```

Reply:
My intended fix was this:

```suggestion
        return !(start._open ?? false) || !(start.content ?? false) ? 0 : 1 + this.getOpenStart(start.content);
```

Using nullish-coallescing to handle the `undefined` and `null`. I didn't like this much when I first did it but it's growing on me, since it semantically 'clicks' for my brain. What do you think?